orttraining/orttraining/python/training/ortmodule/experimental/hierarchical_ortmodule/_hierarchical_ortmodule.py

```diff
@@ -1,15 +1,22 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 # debug_options.py
+import tempfile
 import torch
-from . import ORTModule
+from ... import ORTModule
 
 
-class HierarchalORTModule(torch.nn.Module):
+class HierarchicalORTModule(torch.nn.Module):
+    # This class recursively wraps sub-modules of "self._wrapped_module"
+    # as ORTModule whenever it can. The actual wrapping happens in its
+    # first "forward(...)" call because we need to call Pytorch-to-ONNX
+    # exporter with actual inputs. Afterwards, supported computation
+    # will be delegated to ORT and unsupported computation is still
+    # done by Pytorch.
     def __init__(self, module):
-        super(HierarchalORTModule, self).__init__()
+        super(HierarchicalORTModule, self).__init__()
         self._initialized = False
-        self._module = module
+        self._wrapped_module = module
```

Comment:
```suggestion
        self._original_module = module
```
Use the same terminology as ORTModule for easier understanding. Different mental models may be confusing

Reply:
Ok.